src/test/groovy/edu/ie3/datamodel/io/factory/timeseries/CosmoTimeBasedWeatherValueFactoryTest.groovy

```diff
@@ -72,7 +78,9 @@ class CosmoTimeBasedWeatherValueFactoryTest extends Specification {
         Quantities.getQuantity(282.671997070312d, StandardUnits.SOLAR_IRRADIANCE),
         Quantities.getQuantity(278.019012451172d, StandardUnits.TEMPERATURE),
         Quantities.getQuantity(0d, StandardUnits.WIND_DIRECTION),
-        Quantities.getQuantity(1.66103506088257d, StandardUnits.WIND_VELOCITY)))
+        Quantities.getQuantity(1.66103506088257d, StandardUnits.WIND_VELOCITY),
+        Quantities.getQuantity(278.019012451172d, StandardUnits.TEMPERATURE),
+        Quantities.getQuantity(278.019012451172d, StandardUnits.TEMPERATURE)))
```

Comment:
where does these temperatures coming from, when your input in line 69 - 70 is an empty string